### DISCUSSION TITLE
Embedded video not playing

### DISCUSSION THREAD

**Discussion by dansity** [requarks/wiki#6601] [General]
I have embedded a video with the ``` <video> ``` html code.
```
<video width="720" controls>
 < source src="\\server\test\recording_13.mp4" type="video/mp4">
</video>
```

When I paste the video URL into the browser it is working hence the encoding format is ok. When I load the page with the embed video I see the video player but it is just an empty player, file is not possible to play.
Wikijs is hosted on a VM and that VM also has access to my \\server\test folder

wikijs version 2.5.299
nodejs 12.22.12
Postgresql 12.15

I'm thankful for any suggestion.

**Comment by NGPixel** [comment]
Make sure to disable HTML Sanitization in Admin > Rendering > HTML > Security and re-save your page.

**Reply by dansity** [reply]
Thank you for your answer, video still not loading, it is an empty player.
